contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -131,6 +131,14 @@ Datum		TdsTypeSqlVariantToDatum(StringInfo buf);
 static void FetchTvpTypeOid(const ParameterToken token, char *tvpName);
 
 /* Local structures for the Function Cache by TDS Type ID */
+typedef struct
```

Comment:
This struct should have it's own comment stating that this is copy of a struct from POSTGIS.

Reply:
Done